pkg/sdk/go/channels.go

```diff
@@ -9,6 +9,7 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/absmach/magistrala/internal/apiutil"
```

Comment:
Do the ID check for `ChannelsByThing`, `ChannelPermissions`, `ChannelPermissions`, `ChannelPermissions`,`ListChannelUsers`, `ListChannelUsers`, `ListChannelUsers`, `ListChannelUsers`, `ConnectThing`, `ConnectThing`, (`ConnectThing`, `ConnectThing` -> inside changeChannelStatus)

Reply:
These checks already exist on the api level